src/BlockType.cpp

```diff
@@ -260,7 +260,7 @@ AString ItemTypeToString(short a_ItemType)
 
 AString ItemToFullString(const cItem & a_Item)
 {
-	return fmt::format(FMT_STRING("{}:{} * {}"), ItemToString(a_Item), a_Item.m_ItemDamage, a_Item.m_ItemCount);
+	return fmt::format(FMT_STRING("{}:{:d} * {:d}"), ItemToString(a_Item), a_Item.m_ItemDamage, a_Item.m_ItemCount);
```

Comment:
because it can't be printed correctly in console, because its type is "short", if directly print it, will show non-printable characters. i found this problem in my fixing progress.

Reply:
i test again, "short" type is show correctly, but "char" is still non-printable in console.

```
		char s = 44;
		std::cout << fmt::format("Test short: {}", s);
```

output：
```
[257f70e6af82184b|22:10:05] Waiting for the Deadlock Detector thread to finish
[257f70e6af82184b|22:10:05] The Deadlock Detector thread finished
Test short: ,
PS F:\cuberite\build\Server>
```

Reply:
Right, `m_ItemDamage` is a `short`, but `m_ItemCount` is a `char` so that really got messed up in #5497. Good catch. Still, it probably shouldn't be in this PR, but a separate one (I'll do that), as it's a different issue. Think of going through git blame in a year or two, and wondering, "what does this have to do with minecarts?"